eark_validator/structure.py

```diff
@@ -40,6 +39,7 @@
 
 METS_NAME = 'METS.xml'
 STR_REQ_PREFIX = 'CSIPSTR'
+ROOT = 'root '
```

Comment:
Perhaps it is better to use ROOT without the space at the end and then combine it with f‘{ROOT} {self.name}’